packages/reports/tests/integration/components/report-actions/export-test.js

```diff
@@ -59,7 +59,7 @@ module('Integration | Component | report actions - export', function(hooks) {
           type: 'timeDimension',
           source: 'bardOne',
           field: 'network.dateTime',
-          parameters: { grain: 'day' },
+          parameters: { grain: 'all' },
           operator: 'bet',
           values: ['current', 'next']
```

Comment:
>kevinhinterlong 30 minutes ago Member
Yea change it back to 'day' 👍

yeah, b/c `['current', 'next']` does not work with `all`

Reply:
Got it